docs/testing.rst

```diff
@@ -8,148 +8,176 @@
 
-    PHANTOMJS_EXECUTABLE=./node_modules/.bin/phantomjs ./node_modules/.bin/casperjs
+.. code-block:: bash
 
-You should see output that starts with something similar to::
+    $ grunt test
 
-    CasperJS version 1.1.0-beta3 at /bedrock/node_modules/casperjs, using phantomjs version 1.9.8
+See the `Jasmine`_ documentation for tips on how to write JS behavioral or unit
+tests. We also use `Sinon`_ for creating test spies, stubs and mocks.
 
-Running CasperJS tests
+Running Selenium tests
 ----------------------
 
 .. Note::
 
-  Before running the CasperJS tests, please make sure to follow the bedrock :ref:`installation
-  docs<install>`, including the database sync that is needed to pull in external data such
-  as event/blog feeds etc. These are required for some of the tests to pass.
+  Before running the Selenium tests, please make sure to follow the bedrock
+  :ref:`installation docs<install>`, including the database sync that is needed
+  to pull in external data such as event/blog feeds etc. These are required for
+  some of the tests to pass.
 
-To run the full functional test suite against your local bedrock instance, type::
+To run the full functional test suite against your local bedrock instance:
 
-    npm test -- tests/functional --config=tests/config.json
+.. code-block:: bash
 
-This is just shorthand for the following command::
+    $ py.test --driver Firefox --html results.html -n auto tests/functional/
 
-    PHANTOMJS_EXECUTABLE=./node_modules/.bin/phantomjs ./node_modules/.bin/casperjs test tests/functional --config=tests/config.json
+This will run all test suites found in the ``tests/functional`` directory and
+assumes you have bedrock running at ``localhost`` on port ``8000``. Results will
+be reported in ``results.html`` and tests will run in parallel according to the
+number of cores available.
 
-This will run all test suites found in the ``tests/functional`` directory and assumes you
-have bedrock running at ``localhost`` on port ``8000``.
+.. Note::
 
-To run a single test suite you must tell CasperJS to execute a specific file
-e.g. ``tests/functional/home.js``::
+    For the above command to work, Firefox needs to be installed in a
+    predictable location for your operating system. For details on how to
+    specify the location of Firefox, or running the tests against alternative
+    browsers, refer to the `pytest-selenium documentation`_.
 
-    npm test -- tests/functional/home.js --config=tests/config.json
+To run a single test file you must tell py.test to execute a specific file
+e.g. ``tests/functional/test_newsletter.py``:
 
-You can also easily run the tests against any bedrock environment by specifying the
-``--domain`` variable. For example, to run all functional tests against dev::
+.. code-block:: bash
 
-    npm test -- tests/functional --domain=https://www-dev.allizom.org --config=tests/config.json
+    $ py.test --driver Firefox --html results.html -n auto tests/functional/test_newsletter.py
```

Comment:
When I try to run this line I get the following error:

    usage: py.test [options] [file_or_dir] [file_or_dir] [...]
    py.test: error: unrecognized arguments: -n

If I omit `-n auto` the tests seem to run fine